tests/wpt/web-platform-tests/css/css-backgrounds-3/body-bgcolor-attribute-change.html

```diff
@@ -0,0 +1,9 @@
+<!doctype html>
+<link rel="match" href="body-bgcolor-attribute-change-ref.html" />
```

Comment:
`./mach test-tidy` is failing now:

```
Running the WPT lint on ./tests/wpt/web-platform-tests/...
  Progress: 100% (2/2){"path": "css/css-backgrounds-3/body-bgcolor-attribute-change.html", "message": "Testcase file must have a link to a spec", "lineno": null, "rule": "MISSING-LINK"}
WPT Lint Tool:: lint error(s) in Web Platform Tests: exit status 1
```

I'm not sure how the link is missing, as I've followed the reftests docs, and they passed previously when these tests were in a different folder. Additionally, when I run this test, it passes.

Any idea what's wrong?

Reply:
You need to add:

```html
<link rel="help" href="https://html.spec.whatwg.org/multipage/obsolete.html#attr-body-bgcolor">
```